src/interfaces/chat_app/app.py

```diff
@@ -1062,6 +1166,50 @@ def dislike(self):
             if self.chat.conn is not None:
                 self.chat.conn.close()
 
+    def text_feedback(self):
```

Comment:
maybe to reduce load in this app.py, we can make a utilities file with all the feedback-related functions